frontend/containers/home/use-platform/component.tsx

```diff
@@ -117,7 +118,7 @@ export const UsePlatform = () => {
             <Button
               theme="secondary-green"
               size="small"
-              to="/investors"
+              to={Paths.ForInvestors}
```

Comment:
Unfortunately, this button is not clickable because one of the container has the class `-z-10`. Could you have a look at it?

Reply:
@clementprdhomme  I've tried changing this but I can't get a solution to this z-index problem. If I remove the -z-10 the search box filters are shown behind this section. Could you have a look, please?